packages/amplify_core/lib/src/types/analytics/analytics_types.dart

```diff
@@ -1,8 +1,13 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+/// Exceptions.
+export '../exception/analytics/analytics_exception.dart';
+export '../exception/network_exception.dart';
+export '../exception/unknown_exception.dart';
```

Comment:
As categories now share the same `unknown_exception.dart` should it be exported from `amplift_core.dart` instead of from each category type entry?

Reply:
Thanks updated the exports.